packages/discord.js/src/util/Status.js

```diff
@@ -3,7 +3,7 @@
 const { createEnum } = require('./Enums');
 
 /**
- * @typedef {Object} Status
+ * @typedef {Object<GatewayOpcodes>} Status
```

Comment:
This isn't exactly right as it's missing some members (see https://discord-api-types.dev/api/discord-api-types-v10/enum/GatewayOpcodes).